coinstac-graphql/config/api-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "algo": "sha512",
+  "cstacDB": "coinstac",
+  "encoding": "base64",
+  "hapiPort": 3100,
+  "hashBytes": 64,
+  "host": "localhost",
+  "iterations": 500000,
+  "jwtSecret": "alex",
+  "rethinkPort": 28015,
+  "saltBytes": 16
```

Comment:
This has all changed. Will remove when I pull everything down to get a working version.